css/modal.css

```diff
@@ -0,0 +1,205 @@
+.modal {
+  display: none;
+  position: fixed;
+  z-index: 1000;
+  padding-top: 100px;
+  padding-bottom: 100px;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgb(0, 0, 0);
+  background-color: rgba(0, 0, 0, 0.4);
+}
+
+/* Modal Content */
+.modal-content {
+  background-color: #fefefe;
+  margin: auto;
+  padding: 20px;
+  border: 1px solid #888;
+  width: 70%;
+  border-radius: 10px;
+}
+
+/* The Close Button */
+.close {
+  color: #aaa;
+  float: right;
+  font-size: 28px;
+  font-weight: bold;
+}
+
+.close:hover,
+.close:focus {
+  color: #000;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.desktop-bg {
+  background-size: cover;
+  background-repeat: no-repeat;
+  width: 100%;
+  height: 100%;
+}
+
+.modal-body {
+  padding: 2px 16px;
+  height: 100%;
+}
+
+.modal-body h3 {
+  width: 259px;
+  height: 135px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+#tech {
+  display: none;
+}
+
+#mobile-ul {
+  display: flex;
+  align-items: center;
+  padding: 0;
+  gap: 8px;
+  height: 32px;
+  justify-content: flex-start;
+}
+
+#mobile-ul > li {
+  background: #ebf0ee;
+  border-radius: 4px;
+  align-items: center;
+  padding: 8px 10px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #3a4a42;
+}
+
+#description {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 18px;
```

Comment:
- [ ] Kindly make sure you reduce the **font size** of the project description text and use the one from your template to match your chosen Figma design template